.vscode/settings.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
are you sure you wanted commit this file?

Reply:
Yes. This helps configuring formatter etc. for vscode.